src/test/java/liuni/LiUniResourceTest.java

```diff
@@ -121,7 +123,11 @@ public void testREST_postTweet_successfulPost() {
 
     @Test
     public void testREST_postTweet_badPost_badTweet() {
-        String testString = "This is a bad string because it is going go to be longer than Twitter's limits. It is this long because it is Lorem Ipsum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In vehicula velit vel augue porta condimentum. Nullam ornare velit mattis, maximus sem eget, malesuada nibh. Interdum et malesuada fames ac ante ipsum primis in faucibus. Suspendisse potenti. Pellentesque a dictum lorem. Ut sit amet fringilla turpis. Sed pretium, erat tincidunt aliquam mattis, neque diam luctus mauris, sed lacinia arcu libero sit amet tortor.";
+        StringBuffer stringBuffer = new StringBuffer();
+        for (int i = 0; i < TWITTER_CHAR_MAX + 1; i ++){
```

Comment:
java has a function you can use to generate a string instead of writing one yourself

Reply:
Was under the impression that that was what I was doing, but now see the StringUtils function and will change to using that.